ci/testkomodo.sh

```diff
@@ -92,7 +92,7 @@ run_everest_egg_test() {
     fi
     cat "$CONFIG"
 
-    everest run "$CONFIG"
+    everest run "$CONFIG" --debug
```

Comment:
You will get many debug messages in the console output, but I guess that is what you want.